src/ci/docker/x86_64-gnu/Dockerfile

```diff
@@ -16,5 +16,17 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 COPY scripts/sccache.sh /scripts/
 RUN sh /scripts/sccache.sh
 
-ENV RUST_CONFIGURE_ARGS --build=x86_64-unknown-linux-gnu --enable-sanitizers --enable-profiler
-ENV SCRIPT python2.7 ../x.py test
+ENV HOSTS=x86_64-unknown-linux-gnu
+
+ENV TARGETS=x86_64-unknown-linux-gnu
+ENV TARGETS=$TARGETS,x86_64-unknown-linux-gnux32
```

Comment:
Do you plan on leaving this permanently? It looks like we don't have a lot of extra time on this builder unfortunately (the last one clocked in at 1h40m)...

I'd imagine though that once we get the test suite passing once it's probably pretty unlikely to regress?